Sources/Growth/DAU.swift

```diff
@@ -12,8 +12,16 @@ public class DAU {
   public static let defaultWoiDate = "2016-01-04"
 
   private static let apiVersion = 1
-  private static let baseUrl = "https://laptop-updates.brave.com/\(apiVersion)/usage/ios?platform=ios"
 
+  private static var baseUrl: String {
+    get {
+      let domain = AppConstants.buildChannel == .debug || AppConstants.buildChannel == .enterprise
```

Comment:
this logic is incorrect now.
Let's use prod url for `public` channels - beta, prod.
And staging for everything else

Reply:
Yeah you are right Fix at https://github.com/brave/brave-ios/pull/8194/commits/a78ff180fcc0bc02c35c0dd3f2451e3bdebeecf2